docs/reference/Func/SECValidateAccessToken.md

```diff
@@ -0,0 +1,67 @@
+##### Function : User Registration
+##### SECValidateAccessToken - Validates a signed JWT access token.
+---
+```
+#include <bsafe.h>
+STATUS LNPUBLIC  SECValidateAccessToken(
+
+	const char  *pszAccessToken,
+	const char  *pszProviderURL,
+	const char  *pszRequiredScope,
+	const char  *pszResourceURL,
+	DWORD  dwFlags,
+	void  *vpOptionalParams,
+	DWORD  dwMaxEmailSize,
+	char  *retszEmail,
+	DWORD  *retdwDurationSec);
+```
+
+**Description :**
+
+Validates a signed JWT access token that was generated by a trusted OIDC provider. Intended to be called by multiple different server tasks, including external code.
+
+
+**Parameters :**
+Input :
+*pszAccessToken  -  Points to the B64url encoded signed JWT to validate.
+
+*pszProviderURL  -  Base URL of trusted OIDC provider for this connection.
+
+*pszRequiredScope  -  "Domino.user.all" or equivalent
+
+*pszResourceURL  -  Expected value in audience (aud) claim, such as https://www.example.com.
+
+dwFlags  -  Modify behavior from defaults. See fJWT_validate_* below.
+
+*vpOptionalParams  -  pszCustomClaimName.
```

Comment:
Not clear on this pszCustomClaimName & ResourceCallback in next line

Reply:
Corrected